### ISSUE TITLE
Accessibility audit: (JAWS) Tab component missing expanded states

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#300]
## Describe the issue
Global issue to all views with examples. 
The tab and related panel should have an expanded attribute boolean for its expanded and collapsed states.

## Steps to reproduce the issue
With JAWS on navigate to a view with an example and select and tab and announce the related panel.
 
## Actual vs expected behaviour
There should be a relationship between the tab when selected to the expanded panel the tab controls.

## Fix Recommendation
- [x] Add 'aria-expanded=false' to the tab link and button.

## Related stories

- [Accessibility audit: (JAWs) Nunjucks Tab missing from list item list](https://github.com/dwp/design-system/issues/299)
- [Accessibility audit: (JAWS) Tab component, tab differentiation with hidden text when using multiple examples](https://github.com/dwp/design-system/issues/301)
- [Accessibility audit: (JAWS) Tabs appear as links but missing for link items list](https://github.com/dwp/design-system/issues/302)
- [Accessibility audit: Tabs UI different from GDS Tab component](https://github.com/dwp/design-system/issues/303)
